airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -110,6 +110,7 @@ def unnest(self, records: Iterable[MutableMapping[str, Any]]) -> Iterable[Mutabl
                 f"{top_level_name}_{name}": value for (top_level_name, data) in data_to_unnest.items() for (name, value) in data.items()
             }
             final = {**record, **unnested_data}
+            final.pop("properties", None)  # HACK: Disable field in Airbyte does not work for a moment :(
```

Comment:
Could you clarify the comment here? I see the sense in removing this field given that we are already unnesting the data, but I ran some tests on the production connector and the built-in field disabling does seem to work properly.